src/ggml-alloc.c

```diff
@@ -150,6 +155,7 @@ static void remove_allocated_tensor(struct ggml_dyn_tallocr * alloc, size_t offs
 }
 #endif
 
+// Check with reviewer: could that function returns a ggm_status (offset being an arg)
```

Comment:
This function also does not allocate any (physical) memory, it is just calculating offsets within a buffer. If it fails, it means there is a bug somewhere else.

Reply:
ok, but see it can still abort.

Reply:
The abort is mostly a sanity check, it cannot happen if everything is working as expect. If it fails, it means there is a serious bug in ggml.